src/scripts/OutSystems/OSUI/Utils/Menu.ts

```diff
@@ -3,17 +3,285 @@ namespace OutSystems.OSUI.Utils.Menu {
 	// OrientationChange callback to be stored and removed on Destroy
 	let _onOrientationChangeCallback: OSFramework.OSUI.GlobalCallbacks.Generic;
 
+	// App Properties
+	const _appProp = {
+		device: {
+			isLandscape: false,
+			type: undefined as OSFramework.OSUI.GlobalEnum.DeviceType,
+		},
+		layout: {
+			element: undefined as HTMLElement,
+			isAsideExpandable: false,
+			isAsideMenu: false,
+			isAsideMenuOverlay: false,
+			isBlank: false,
+			isTopMenu: false,
+		},
+		menu: {
+			element: undefined as HTMLElement,
+			hasEventListeners: false,
+			isOpen: false,
+		},
+	};
+
+	// Focusable elements on the menu context sine it's different from the default constant that excludes disabled elements with tabindex=-1
```

Comment:
```suggestion
	// Focusable elements on the menu context since it's different from the default constant that excludes disabled elements with tabindex=-1
```